apps/juxtaposition-ui/src/config.js

```diff
@@ -6,6 +6,8 @@ const schema = z.object({
 	logFolder: z.string().default(`${__dirname}/../logs`),
 	httpPort: z.coerce.number().default(8080),
 	httpCookieDomain: z.string().default('.pretendo.network'),
+	metricsEnabled: z.coerce.boolean().default(false),
```

Comment:
unfortunately since its coming from environment variable `z.coerce.boolean()` doesnt work.
You can use `zodCoercedBoolean()` from `@neato/config` instead. See the docs here for details: https://neatojs.com/docs/config/api/utils#zod-coerced-boolean

also applies to miiverse-api/src/config.ts